docsteady/__init__.py

```diff
@@ -220,6 +221,19 @@ def generate_report(format, username, password, plan, path):
 
     print(_as_output_format(appendix_text), file=appendix_file)
 
+    if trace:
```

Comment:
Does this actually do the right thing?  The code seems like an exact copy of the above, so it would seem like it would get the same template and fill it out slightly differently.  If it is in fact correct, then it would seem that much of the duplicated code could be eliminated.

Reply:
The above block has to be removed.